test/e2e/pkg/testnet.go

```diff
@@ -73,7 +73,7 @@ type Node struct {
 	StartAt          int64
 	FastSync         string
 	StateSync        bool
-	ABCIProtocol     Protocol
+	ABCIProtocol     Protocol // always set to builtin in this repository
```

Comment:
As I understand, ABCI is an API bridge between any custom state machine and consensus layer, which can also run as a separate process/machine over UNIX/TCP/GRPC. Why are we removing the support for that and relying only on built-in?

Reply:
Because the only purpose of this repository will be to run the lazyledger-app which will be in process (builtin).